homeassistant/components/ozw/services.py

```diff
@@ -138,3 +146,10 @@ def async_remove_node(self, service):
         instance_id = service.data[const.ATTR_INSTANCE_ID]
         instance = self._manager.get_instance(instance_id)
         instance.remove_node()
+
+    @callback
+    def async_cancel_command(self, service):
+        """Tell the controller to cancel an add or remove command."""
+        instance_id = service.data[const.ATTR_INSTANCE_ID]
+        instance = self._manager.get_instance(instance_id)
+        instance.cancel_controller_command()
```

Comment:
Do we need to guard for a missing instance, ie None value?

Reply:
Probably not a bad idea. Right now if you pass an invalid instance_id it fails with `AttributeError: 'NoneType' object has no attribute 'cancel_controller_command'`

What's the best way to fail a service call gracefully and send a "friendly" message to the user?

Reply:
Raise an error, eg `ValueError` since the user passed an invalid value.

Reply:
Done :smiley: